codex-rs/mcp-server/src/codex_message_processor.rs

```diff
@@ -244,6 +263,61 @@ impl CodexMessageProcessor {
     }
 }
 
+async fn apply_bespoke_event_handling(
```

Comment:
What is "bespoke"?

Reply:
Well, as opposed to the general treatment we give to apply `EventMsg` variants in `add_conversation_listener()`, which is that we forward all of them as notifications, whether or not we believe they are actually useful.

Though as noted in:

```rust
// For now, we send a notification for every event,
// JSON-serializing the `Event` as-is, but we will move
// to creating a special enum for notifications with a
// stable wire format.
```

I would like to stop doing that in the near future.